src/citationnet/static/pieChart_vega-lite.js

```diff
@@ -0,0 +1,88 @@
+const forChartDiv = document.getElementById("forChartDiv");
+const forChartDivvis = document.getElementById("forChartDivvis");
+const forChartDivloader = document.getElementById("forChartDivloader");
+
+dragElement(forChartDiv);
+
+function toggleFORChart() {
+    if (forChartDiv.hidden) {
+        showFOR();
+    } else {
+        hideFOR();
+    }
+}
+
+async function showFOR() {
+    document.getElementById("btnFORChart").style.fontWeight = "bolder";
+    forChartDivvis.hidden = true;
+    forChartDivloader.hidden = false;
+    forChartDiv.hidden = false;
+
+    await loadFOR();
+
+    forChartDivvis.hidden = false;
+    forChartDivloader.hidden = true;
+}
+
+function hideFOR() {
```

Comment:
If you end up attaching click events in `loadFOR`, make sure to remove them here.